content/creativecommons/step2.md

```diff
@@ -0,0 +1,25 @@
+---
+date: "2019-08-02"
+title: "Creative Commons"
+description: ""
+showOnFrontPage: false
+---
+
+<content>
+
+## Short Introduction to Creative Commons
+<youtube>
+https://youtu.be/oLcVycFkmtM
```

Comment:
Disse youtube-lenkene har feil format. Det må være på
`https://www.youtube.com/watch?v={id}`

Så her må det f.eks endres fra `https://youtu.be/oLcVycFkmtM` til `https://www.youtube.com/watch?v=oLcVycFkmtM`